app/views/shared/analytics/_control_rate_panel.html.erb

```diff
@@ -11,6 +11,8 @@
              options: { id: 'control-rate-graph' },
              column_class: 'featured-graph-column',
              graph_data: control_rate_per_month
-                 .map { |month, no_of_patients| [month.strftime('%b'), no_of_patients] }.to_h,
+                 .select { |month, _| month >= number_of_months.months.ago.at_beginning_of_month }
```

Comment:
This could simply be a helper like `control_rate_per_month_panel_data` or something akin.

Reply:
Using `latest_months` helper here.